inc/fieldVariables.php

```diff
@@ -26,7 +26,7 @@
         'label' => 'Icon',
         'name' => 'icon',
         'type' => 'text',
-        'instructions' => 'Enter a valid icon name from Feather Icons (e.g. `check-circle`). Visit https://feathericons.com/ to see the all icons with their corresponding names.',
+        'instructions' => 'Enter a valid icon name from <a href="https://feathericons.com">Feather Icons</a> (e.g. `check-circle`).',
```

Comment:
How about adding target blank? Because if I click it directly while editing, my content changes will be lost. Or is WordPress warning anyways and we'll leave it to the user?